deployment/ccip/changeset/v1_6/cs_home_chain.go

```diff
@@ -667,3 +690,301 @@ func removeNodesLogic(env deployment.Environment, c RemoveNodesConfig) (deployme
 	env.Logger.Infof("Created proposal to remove nodes")
 	return deployment.ChangesetOutput{MCMSTimelockProposals: []mcmslib.TimelockProposal{*proposal}}, nil
 }
+
+// AddOrUpdateNopsConfig is the configuration for adding, updating or removing node operators
+// For adding or updating node operators, ExistingNops should be empty
+// For removing node operators, NopUpdates should be empty
+// For updating node operators, ExistingNops should contain the existing node operators and
+// NopUpdates should contain the node operators to update, it's a map of the existing node operator name to the new node operator
+//
+// For example,
+//
+// when updating nops -
+// ExistingNops: [ { Name: "nop1", Admin: "0x123" }, { Name: "nop2", Admin: "0x456" } ]
+// NopUpdates: { "nop1": { Name: "nop1New", Admin: "0x345" }, "nop2": { Name: "nop2New", Admin: "0x789" } }
+//
+// For adding nops -
+// ExistingNops: []
+// NopUpdates: { "nop1": { Name: "nop1", Admin: "0x123" }, "nop2": { Name: "nop2", Admin: "0x456" } }
+//
+// For removing nops -
+// ExistingNops: [ { Name: "nop1", Admin: "0x123" }, { Name: "nop2", Admin: "0x456" } ]
+// NopUpdates: []
+type AddOrUpdateNopsConfig struct {
+	HomeChainSel uint64
```

Comment:
Wondering if we shouldn't prompt for this & instead infer it from environment ...